app/design/frontend/Clothes/touch/Magento_Cms/layout/cms_index_index.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details
+ */
+-->
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <body>
+        <referenceContainer name="main.content">
+            <container name="mag11.containers.one" htmlTag="div" htmlClass="mag11-containers"  htmlId="mag11-containers-one"></container>
```

Comment:
@yuratouch If block or container is empty, it should have `< />` instead of `< >< />`